test/models/user_test.rb

```diff
@@ -0,0 +1,235 @@
+require "test_helper"
+
+describe User do
+  it "can be instantiated" do
+    assert(users(:new_user).valid?)
```

Comment:
This is an assert-style test.  It works, but it's not the style of test we're using.

```suggestion
    expect(users(:new_user)).must_be :valid?
```